frontend/src/components/Home.js

```diff
@@ -24,7 +24,7 @@ class Home extends React.Component {
       lng: -97,
       lat: 38,
       zoom: 3.7,
-      isSidebarOpen: true,
+      isOpen: true,
```

Comment:
wait, why is this being changed back to `isOpen`? this should be aligned with `master` instead of an old version of megha's branch

Reply:
Got it.